--- a/openldap-2.4.40/libraries/liblutil/Makefile.in	2014-01-25 14:36:15.000000000 +0100
+++ b/openldap-2.4.40/libraries/liblutil/Makefile.in	2014-06-18 10:38:48.142011093 +0200
@@ -16,6 +16,8 @@
 LIBRARY	= liblutil.a
 PROGRAM = testavl
 
+SHAREDLIB = libldif.la
+
 LDAP_INCDIR= ../../include       
 LDAP_LIBDIR= ../../libraries
 
@@ -27,14 +29,14 @@ UNIX_OBJS = detach.o
 
 XLIBS = $(LIBRARY) $(LDAP_LIBLBER_LA)
 
-SRCS	= base64.c entropy.c sasl.c signal.c hash.c passfile.c \
+SRCS	= ldif.c base64.c entropy.c sasl.c signal.c hash.c passfile.c \
 	md5.c passwd.c sha1.c getpass.c lockf.c utils.c uuid.c sockpair.c \
 	avl.c tavl.c \
 	testavl.c \
 	meter.c \
 	@LIBSRCS@ $(@PLAT@_SRCS)
 
-OBJS	= base64.o entropy.o sasl.o signal.o hash.o passfile.o \
+OBJS	= ldif.o base64.o entropy.o sasl.o signal.o hash.o passfile.o \
 	md5.o passwd.o sha1.o getpass.o lockf.o utils.o uuid.o sockpair.o \
 	avl.o tavl.o \
 	meter.o \
@@ -59,3 +61,21 @@ slapdmsg.res: slapdmsg.rc slapdmsg.bin
 clean-local:
 	$(RM) *.res
 
+SHAREDLIBOBJS = ldif.lo
+
+all-local: $(SHAREDLIB)
+
+.SUFFIXES: .c .o .lo
+
+.c.lo:
+	$(LTCOMPILE_LIB) $<
+
+$(LIBRARY): $(SHAREDLIBOBJS) version.lo
+
+$(SHAREDLIB): $(SHAREDLIBOBJS) version.lo
+	$(LTLINK_LIB) -o $(SHAREDLIB) $(SHAREDLIBOBJS) version.lo $(LINK_LIBS)
+
+install-local: FORCE
+	-$(MKDIR) $(DESTDIR)$(libdir)
+	$(LTINSTALL) $(INSTALLFLAGS) -m 644 $(SHAREDLIB) $(DESTDIR)$(libdir)
+	$(LTFINISH) $(DESTDIR)$(libdir)
--- a/openldap-2.4.40/libraries/liblutil/ldif.c	1970-01-01 01:00:00.000000000 +0100
+++ b/openldap-2.4.40/libraries/liblutil/ldif.c	2014-06-18 10:36:19.332696175 +0200
@@ -0,0 +1,4 @@
+#include <ldap.h>
+
+void _shim() {}
+
